frontend/modules/shared/ui/src/lib/layout/footer/footer.component.html

```diff
@@ -0,0 +1,2 @@
+<img src="assets/images/logo_reunice.png" alt="logo reunice" />
```

Comment:
Trochę lepszy alt text by sie przydał, może "EUNICE - European University of Customised Education"